README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+This interface handles relation with [grafana agent charm][grafana-agent],
+and by extension, integration with [COS][cos].
+
+Grafana Agent charm provides [cos agent library][cos-agent] that implements
+both sides of this relation. However, the library is compatible only with Operator
+charms. This interface layer implements "provides" side of the same interface and
+allows reactive charms to relate with Grafana Agent.
+
+## Dependency notice
+
+It seems that when charmcraft builds a charm, it ignores dependencies defined
+in interface's `wheelhouse.txt`. Therefore, if you want to add this interface to
```

Comment:
charmcraft uses the original [charm-tools](https://github.com/juju/charm-tools) to build reactive charms.

So I don't think `wheelhouse.txt` has ever been processed for interfaces.

Should we perhaps just mention the dependency the charm author needs to add here and drop the interface `wheelhouse.txt`?

Reply:
Yes, I'll definitely update this section in readme. The current form was pretty much a question for reviewers :D

However I wouldn't drop the `wheelhouse.txt` file. We need to install our direct dependencies for unit tests, so I'd rather keep the file as the one source of truth and refer to it from documentation. Perhaps we could rename it to more standard `requirements.txt`, since keeping the `wheelhouse` name doesn't give us any benefits.